assets/.storybook/main.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     '@storybook/addon-links',
     '@storybook/addon-essentials',
     '@storybook/addon-interactions',
+    '@storybook/preset-scss',
```

Comment:
I was trying to get our scss modules working in storybook, but didn't have a lot of luck there. I want to re-visit this eventually